src/app/sketch-manager/sketch-manager.component.ts

```diff
@@ -75,6 +65,10 @@ export class SketchManagerComponent implements OnInit, AfterViewInit, OnDestroy
       alert('Save changes to sketch');
       return;
     }
+    const navigationExtras: NavigationExtras = {
```

Comment:
bine ca ai zis, motivul este pt ca redirect to root va trebui sa pointeze inspre /sketches, iar asta tre sa duca inspre /dashboard asa cum e acuma, deci is pathuri diferite